kit-de-demarrage/how_to.md

```diff
@@ -0,0 +1,160 @@
+# How to
```

Comment:
Il faut mettre à jour le README.MD pour la section How To du coup et ajouter un lien

Reply:
Done